dir.targets

```diff
@@ -85,4 +84,11 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="ReportConfigurationErrorMessage"
+          BeforeTargets="ResolveProjectReferences"
+           Condition="'$(BuildConfigurations)' != ''" >
+    <Message Importance="High" Text="$(MSBuildProjectFullPath), Configuration: $(Configuration)" />
```

Comment:
In my local changes I have also been dumping `$(BuildConfiguration)` and `$(BuildConfigurations)` to help with debugging it might be worth adding those and we can keep them until we get this all worked out and then we can turn them to Importance=low